src/main/java/com/bieganski/ox/Board.java

```diff
@@ -0,0 +1,49 @@
+package com.bieganski.ox;
+
+import java.util.TreeSet;
+
+class Board {
```

Comment:
Notice please, that you have 2 responsibilities here. There are 2 (two) reasons for you to change that class. When printing of your board should happen and when you will be changing the way of filling the fields out. That's not that great, from SRP perspective.

Reply:
Hi. Yesterday I had method generating string with board in BoardPrinter class but you have told me that something like that should be moved to toString() method in Board class. Now you are telling us that having it in Board class is breaking SRP, so we are both a bit confused. Could you explain?